jobs/hurricane/stats/JEVS_HURRICANE_STATS

```diff
@@ -1 +1,134 @@
+export NET=${NET:-evs}
+export COMPONENT=${COMPONENT:-define_in_ecf}
+export RUN=${RUN:-define_in_ecf}
+export STEP=${STEP:-stats}
+
+####################################
+# Determine Job Output Name on System
+####################################
+export pgmout="OUTPUT.$$"
+export pgmerr=errfile
+
+####################################
+# SENDECF  - Flag Events on ecFLOW
+# SENDCOM  - Copy Files From TMPDIR to $COMOUT
+# SENDDBN  - Issue DBNet Client Calls
+####################################
+export SENDECF=${SENDECF:-NO}
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-NO}
+
+####################################
+# Specify Execution Areas
+####################################
+export HOMEevs=${HOMEevs:-${PACKAGEHOME}}
+export SCRIPTSevs=${SCRIPTSevs:-$HOMEevs/scripts/hurricane/stats}
+export USHevs=${USHevs:-$HOMEevs/ush/hurricane/stats}
+export PARMevs=${PARMevs:-$HOMEevs/parm/metplus_config/hurricane/stats}
+export FIXevs=${FIXevs:-$HOMEevs/fix}
+
+# Run setpdy and initialize PDY variables
+##############################
+#setpdy.sh
+#. ./PDY
+
+##############################################
+# Define COM directories
+##############################################
+if [ ${COMPONENT} = "hurricane_global_det" -a ${RUN} = "tropcyc" ]; then
+setpdy.sh
+. ./PDY
+export YYYY=`echo ${PDY} | cut -c1-4`
+export YY22=`echo ${PDY} | cut -c3-4`
+
+export COMINvit=${COMINvit:-/your/TC/vitals/file}
+export COMINtrack=${COMINtrack:-your/TC/track/file}
+export COMINbdeckNHC=${COMINbdeckNHC:-/your/NHC/bdeck/data/dir}
+export COMINbdeckJTWC=${COMINbdeckJTWC:-/your/JTWC/bdeck/data/dir}
+
+elif [ ${COMPONENT} = "hurricane_global_ens" -a ${RUN} = "tropcyc" ]; then
+setpdy.sh
+. ./PDY
+export YYYY=`echo ${PDY} | cut -c1-4`
+export YY22=`echo ${PDY} | cut -c3-4`
+
+export COMINvit=${COMINvit:-/your/TC/vitals/file}
+export COMINtrack=${COMINtrack:-your/TC/track/file}
+export COMINbdeckNHC=${COMINbdeckNHC:-/your/NHC/bdeck/data/dir}
+export COMINbdeckJTWC=${COMINbdeckJTWC:-/your/JTWC/bdeck/data/dir}
+
+elif [ ${COMPONENT} = "hurricane_global_det" -a ${RUN} = "tcgen" ]; then
+#setpdy.sh
+#. ./PDY
+export PDY=20211115
+
+export YYYY=`echo ${PDY} | cut -c1-4`
+#export YY22=`echo ${PDY} | cut -c3-4`
+export COMINgenesis=${COMINgenesis:-/your/TC/genesis/data/dir}
+export COMINadeckNHC=${COMINadeckNHC:-/your/NHC/adeck/data/dir}
+export COMINbdeckNHC=${COMINbdeckNHC:-/your/NHC/bdeck/data/dir}
+export COMINadeckJTWC=${COMINadeckJTWC:-/your/JTWC/adeck/data/dir}
+export COMINbdeckJTWC=${COMINbdeckJTWC:-/your/JTWC/bdeck/data/dir}
+
+elif [ ${COMPONENT} = "hurricane_regional" -a ${RUN} = "tropcyc" ]; then
+setpdy.sh
+. ./PDY 
+export YYYY=`echo ${PDY} | cut -c1-4`
+export YY22=`echo ${PDY} | cut -c3-4`
+
+export COMINvit=${COMINvit:-/your/TC/vitals/file}
+export COMINtrack=${COMINtrack:-your/TC/track/file}
+export COMINbdeckNHC=${COMINbdeckNHC:-/your/NHC/bdeck/data/dir}
+export COMINbdeckJTWC=${COMINbdeckJTWC:-/your/JTWC/bdeck/data/dir}
+
+fi
+
```

Comment:
This entire if-elif-fi block is good! But I think we need one final catch to allow the job to exit gracefully should none of the cases be satisfied. This is to catch the off chance for a user typo, for example, in the ecf script when defining $COMPONENT and $RUN.